docs/cmdline-opts/mptcp.md

```diff
@@ -0,0 +1,41 @@
+---
+c: Copyright (C) Dorian Craps, <dorian.craps@student.vinci.be>
+SPDX-License-Identifier: curl
+Long: mptcp
+Added: 8.9.0
+Help: Enable Multipath TCP (MPTCP)
```

Comment:
Since the command line option itself is called `mptcp` maybe we can drop the `(MPTCP)` part from the help text? It seems superfluous to me.

Reply:
Besides that, I think we are ready to merge

Reply:
Thank you for the review!

I just removed the non-necessary `(MPTCP)` in the help text.

I rebased the branch on top of master and force-pushed.

Reply:
(I also had to modify the same help message in `src/tool_listhelp.c`.)